faiss/gpu/GpuIndex.cu

```diff
@@ -236,7 +237,7 @@ void GpuIndex::search(
 
     bool usePaged = false;
 
-    if (getDeviceForAddress(x) == -1) {
+    if (!config_.use_raft && getDeviceForAddress(x) == -1) {
```

Comment:
`raft::neighbors::ivf_pq::search` does internal batching  for large batch size. This way the temporary buffers will be kept under control. But a comment above (in line 226) says
> The input vectors may be too large for the GPU

If that is the motivation to do paging, then we should keep that even when raft is enabled.